### DISCUSSION TITLE
conversionPartLoad subclass: discretization variables spelling inconsistency

### DISCUSSION THREAD

**Discussion by d00958** [FZJ-IEK3-VSA/FINE#60] [Q&A]
Hi all,
in my recent experiments with the conversion subclasses, I have come across what appears to be a small inconsistency in the `conversionPartLoad` subclass.

On a side note: the discretization variables `discretizationPointVariablesOptimun`, `discretizationSegmentConVariablesOptimun`, and `discretizationSegmentBinVariablesOptimun` are all spelled with an "n" at the end. This is **not** what I am referring to as this is fully consistent within the subclass and does not cause an issue.

However, when calling `writeOptimizationOutputToExcel()`, I got an error saying:
> AttributeError: 'ConversionPartLoadModel' object has no attribute '_discretizationPointVariablesOptimun'

As it turns out, the aforementioned discretization variables are all initialized and set without an underscore, but `getOptimalValues()` tries to retrieve them as `_discretizationPointVariablesOptimun`. Eliminating the underscore seems to have solved this for me.

I am not sure if this is relevant or if `conversionPartLoad` plays a role in further development, but maybe someone else faces the same. Also, an option to combine `conversionPartLoad` and `conversionDynamic` would be a dream come true (if even mathematically feasible).

**Comment by d00958** [comment]
For the sake of completeness, sheet names also need to be truncated for conversionPartLoad. See also discussion #58 & issue #59

**Comment by JohannesBehrens** [comment]
Issue is solved in da6416288993b69cc1e7002b2eefe6008ab78257
